source/FreeRTOS_Routing.c

```diff
@@ -291,10 +291,10 @@ void FreeRTOS_FillEndPoint( NetworkInterface_t * pxNetworkInterface,
             else
         #endif
         {
-            FreeRTOS_printf( ( "FreeRTOS_AddEndPoint: MAC: %02x-%02x IPv4: %lxip\n",
+            FreeRTOS_printf( ( "FreeRTOS_AddEndPoint: MAC: %02x-%02x IPv4: %xip\n",
                                pxEndPoint->xMACAddress.ucBytes[ 4 ],
                                pxEndPoint->xMACAddress.ucBytes[ 5 ],
-                               FreeRTOS_ntohl( pxEndPoint->ipv4_defaults.ulIPAddress ) ) );
+                               ( unsigned int ) FreeRTOS_ntohl( pxEndPoint->ipv4_defaults.ulIPAddress ) ) );
```

Comment:
Please change into :

~~~
    FreeRTOS_printf( ( "FreeRTOS_AddEndPoint: MAC: %02x-%02x IPv4: %xip\n",
                        pxEndPoint->xMACAddress.ucBytes[ 4 ],
                        pxEndPoint->xMACAddress.ucBytes[ 5 ],
                        ( unsigned ) FreeRTOS_ntohl( pxEndPoint->ipv4_defaults.ulIPAddress ) ) );
~~~
Thank you